kyakusahmed/EPIC-EMAIL
# Ft get sent messages #165113223

### What does this PR do?
get sent messages

### Description of Task to be completed?
Using java script fetch, a user can now successfully view all his sent messages.

### How should this be manually tested?
To test if the user is able to view his sent messages, on click on sent link, he should be directed straight ti his sent messages

Important Notes
[gh-pages](https://kyakusahmed.github.io/EPIC-EMAIL/templates/signup.html)
@bekeplar @jenny @neelxie @nanfuka @Wabuluka
